dbt/include/clickhouse/macros/materializations/distributed_table.sql

```diff
@@ -98,3 +107,13 @@
   {{ partition_cols(label="partition by") }}
   {{ adapter.get_model_settings(model) }}
 {%- endmacro %}
+
+{% macro create_distributed_shard_table(distributed_relation, shard_relation, structure_relation, sql_query=none) -%}
```

Comment:
nit -- maybe we should call this distributed_local table to match the local prefix?